### PR TITLE
Docs: The Plugins `harbor` And `jenkins` Customize Image Repository Support

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: Daniel Hu <tao.hu@merico.dev>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.

## Description

docs: the plugins `harbor` and `jenkins` customize image repository support

## Related Issues

#1084

## New Behavior (screenshots if needed)

Documents updated only

### CODE REVIEW

File: docs/plugins/harbor.zh.md [hunk 1]
```diff
@@ -395,3 +395,98 @@ harbor-ingress   nginx   core.harbor.domain   192.168.49.2   80      2m8s
...
+      image:
+        repository: [[ imageRepo ]]/goharbor/redis-photon
+        tag: v2.5.3
+  exporter:
+    image:
+      repository: [[ imageRepo ]]/goharbor/harbor-exporter
+      tag: v2.5.3
+```
+
+这段配置中留了一个变量 `[[ imageRepo ]]`，你可以在[变量配置](../../core-concepts/variables.zh)中定义这个变量，变量值设置成你的镜像仓库地址，例如：
```
[the remark this answers is not in the document]
Comment (daniel-hutao) [reply docs/plugins/harbor.zh.md:467]: Yeah, I agree with you.

File: docs/plugins/jenkins.zh.md [hunk 2]
```diff
@@ -334,9 +334,62 @@ workflows: |
...
+imageRepo: harbor.example.com:9000
+```
+
+当然，你需要保证需要的镜像都在你的镜像仓库中存在。
+
+你可以下载[镜像列表文件](./jenkins/jenkins-images.txt)，
+然后借助["Image Pull Push"](https://raw.githubusercontent.com/devstream-io/devstream/main/hack/image-pull-push.sh)工具脚本来准备镜像。
+
+```shell
+curl -o jenkins-images.txt https://docs.devstream.io/en/latest/plugins/jenkins/jenkins-images.txt
```
[the remark this answers is not in the document]
Comment (daniel-hutao) [reply docs/plugins/jenkins.zh.md:382]: I'll update the link here right away.

File: docs/plugins/harbor.zh.md [hunk 3]
```diff
@@ -395,3 +395,98 @@ harbor-ingress   nginx   core.harbor.domain   192.168.49.2   80      2m8s
...
+imageRepo: harbor.example.com:9000
+```
+
+当然，你需要保证需要的镜像都在你的镜像仓库中存在。
+
+你可以下载[镜像列表文件](./harbor/harbor-images.txt)，
+然后借助["Image Pull Push"](https://raw.githubusercontent.com/devstream-io/devstream/main/hack/image-pull-push.sh)工具脚本来准备镜像。
+
+```shell
+curl -o harbor-images.txt https://docs.devstream.io/en/latest/plugins/harbor/harbor-images.txt
```
[the remark this answers is not in the document]
Comment (daniel-hutao) [reply docs/plugins/harbor.zh.md:479]: I'll update the link here right away.

**Review by iyear — APPROVED** [review]
LGTM 👏

I have reviewed docs, but I can't review code about shell script.

**Review by aFlyBird0 — APPROVED** [review]

**Pull request merged**